GlimpseImputationPipeline/Glimpse2MergeBatches.wdl

```diff
@@ -52,20 +52,19 @@ workflow Glimpse2MergeBatches {
 
     # This handles the qc_metrics in the case of only one batch. If qc_metrics is not
     # defined then this if statement will be false and qc_metrics_1 will be null.
-    # If qc_metrics is defined then set qc_metrics_1 to the first element, which
-    # may or may not be null.
+    # If qc_metrics is defined then set qc_metrics_1 to the first element
     if (length(imputed_vcfs) == 1 && defined(qc_metrics)) {
-        File? qc_metrics_1 = select_first([qc_metrics, []])[0]
+        File qc_metrics_1 = select_first([qc_metrics, []])[0]
```

Comment:
since you're requiring qc_metrics be defined here, you don't need the extra empty array, do you?  ie, `select_first([qc_metrics])[0]` should work, right?